template/_dynamic_files.jinja2

```diff
@@ -785,6 +785,73 @@ jobs:
           file: "../../edit-comment.md"
           identifier: "UNREASONED"
 {% endif -%}
+{% if (('github_actions' in project.ci) and ('release-diff' in project.workflows)) %}
+^^^ .github/workflows/release-diff.yml
+name: 'Go Go Release Diff'
+
+on:
+  # Triggers the workflow on pull request events for the master branch
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  release_file:
+    runs-on: ubuntu-latest
+    container: obolibrary/odklite:{% if env is defined -%}{{env['ODK_VERSION'] or "v1.2.32" }}{%- else %}v1.2.32{% endif %}
+    steps:
+      - uses: khan/pull-request-comment-trigger@v1.1.0
+        id: check
+        with:
+          trigger: {% raw %}'#gogoreleasediff'{% endraw %}
+        env:
+          GITHUB_TOKEN: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
+      - uses: actions/checkout@v3
+      # Checks-out main branch under "main" directory
+      - uses: actions/checkout@v3
+        with:
+          ref: master
+          path: master
+      - run: export ROBOT_JAVA_ARGS=-Xmx6G; robot diff --labels True --left master/{{ project.id }}.owl --right {{ project.id }}.owl -f markdown -o release-diff.md
```

Comment:
@matentzn is this the correct approach for checking release diff?

Reply:
yes :)

Reply:
No, this is part of the release process.